CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+### 3.2.2 - [#49](https://github.com/openfisca/country-template/pull/49)
+
+* Tax and benefit system evolution.
+* Impacted periods: all.
+* Impacted areas: `taxes`
+* Details:
+  - Implement housing tax minimal amount
+
+<!-- -->
```

Comment:
Because this PR does two independent things from the user point of view.

(Not so much from our developper point of view, as the whole point of adding the minimal amount for the housing tax is to have a yaml file with several parameters that we can use for test)